config/initializers/carrierwave.rb

```diff
@@ -13,7 +13,7 @@
     config.storage = :fog
     config.cache_dir = Rails.root.join('tmp', 'uploads').to_s
     config.fog_credentials = {
-        provider:              'AWS',
+        provider:              AppConfig.environment.s3.provider.get,
```

Comment:
Layout/FirstHashElementIndentation: Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.